### PR TITLE
refactored populate.js to be more modularity

### PR DESCRIPTION
**Pull request by cthao309**
I refactored my random generator and db query in populate.js to be more modular. Can some review and approve when they got the chance.

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ app.get('/api/restaurants', (req, res) => {
     if(err) {
       res.status(500).send(err);
     } else {
-      console.log('(/api/restaurants) => ', data)
+      console.log('(/api/restaurants) => ', data.length)
```
Comment (serviolee) [inline server/app.js:17]: Should this be res.length?

File: server/app.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ app.get('/api/users', (req, res) => {
     if(err) {
       res.status(500).send(err);
     } else {
-      console.log('(/api/users) => ', data)
+      console.log('(/api/users) => ', data.length)
```
Comment (serviolee) [inline server/app.js:28]: Should this be res.length?

File: server/app.js [hunk 3]
```diff
@@ -36,7 +36,7 @@ app.get('/api/posts', (req, res) => {
     if(err) {
       res.status(500).send(err);
     } else {
-      console.log('(/api/posts) => ', data)
+      console.log('(/api/posts) => ', data.length)
```
Comment (serviolee) [inline server/app.js:39]: Should this be res.length?

File: server/app.js [hunk 4]
```diff
@@ -47,7 +47,7 @@ app.get('/api/postImages', (req, res) => {
...
```
Comment (serviolee) [inline server/app.js:50]: Should this be res.length?

Comment (cthao309) [reply server/app.js:50]: thanks @serviolee

**Pull request merged**